cumulusci/robotframework/tests/salesforce/pageobjects/objectmanager.robot

```diff
@@ -0,0 +1,36 @@
+*** Settings ***
+
+Resource        cumulusci/robotframework/Salesforce.robot
+Library         cumulusci.robotframework.PageObjects
+...  ${CURDIR}/ObjectManagerPageObject.py
+
+
+Suite Setup     Run keywords  Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Keywords ***
+Create Customfield In Object Manager
+    [Documentation]        Reads key value pair arguments.
```

Comment:
Personally I feel that long lines are hard to read. What do you think about removing some of the whitespace? Ultimately it's a judgement call so I'll let you have the final word. Here's how I would format this to make it easier to read:

```
Create Custom field In Object Manager
    [Documentation]
    ...  Reads key value pair arguments.
    ...
    ...  Navigates to Object Manager page and load fields and relationships for the specific object
    ...  Runs keyword to create custom field
    ...
    ...  Example:
    ...
    ...  | Create custom field in object manager
    ...  | ...  Object=Payment
    ...  | ...  Field_Type=Formula
    ...  | ...  Field_Name=Is Opportunity From Prior Year
    ...  | ...  Formula=YEAR( npe01__Opportunity__r.CloseDate ) < YEAR( npe01__Payment_Date__c )
```